test/Microsoft.DocAsCode.MarkdigEngine.Tests/MarkdigServiceTest.cs

```diff
@@ -64,19 +64,5 @@ public void MarkdigServiceTest_ParseAndRender_Inclusion()
             var expectedDependency = new List<string> { "~/x/b/linkAndRefRoot.md" };
             Assert.Equal(expectedDependency.ToImmutableList(), mr.Dependency);
         }
-
-        [Fact]
-        [Trait("Related", "MarkdigService")]
-        public void MarkdigServiceTest_ParseInline()
```

Comment:
>MarkdigServiceTest_ParseInline [](start = 20, length = 30)

why remove this test case?

Reply:
I'll add back the `isInline` interface and revert the change